content/applications/websites/ecommerce/managing_products/price_management.rst

```diff
@@ -0,0 +1,204 @@
+================
+Price management
+================
+
+Odoo offers multiple options to select the price displayed on your website, as well as
+condition-specific prices based on selected criteria.
+
+Displayed price
+===============
+
+Choosing the displayed tax usually depends on a country's regulations or the type of customers
+**(B2B vs B2C)**. To select the type of price displayed, go to :menuselection:`Website app -->
+Configuration --> Settings`, scroll-down to the :guilabel:`Shop - Products` category, and select
+:guilabel:`Tax Excluded` or :guilabel:`Tax Included`.
+
+Note that this setting is **global**, and the tax-display type is the same for your website(s) and
+back-end. It is therefore not possible to select different tax displays for different websites. This
+may be a significant point of consideration when implementing a database with multiple ecommerce
+websites pertaining to different types of customers (i.e. B2B and B2C).
+
+- :guilabel:`Tax Excluded`: the price displayed on the website is **tax-excluded** and the tax is
+  computed at the cart-review step;
+- :guilabel:`Tax Included`: the price displayed on the website is **tax-included**.
+
+You can choose to display the type of pricing next to the product price, by going to
+:menuselection:`Website app --> Site --> Homepage --> Shop`, selecting one of your product, and then
+:menuselection:`Edit --> Customize tab` and enabling :guilabel:`Tax Indication`.
+
+.. image:: price_management/price-tax-display-type.png
+   :align: center
+   :alt: Tax type displayed on the product page
+
```

Comment:
```suggestion

Note that this setting is **global**, and the tax-display type is the same for your website(s) and
back-end. It is, therefore, not possible to select different tax displays for different websites. This
may be a significant point of consideration when implementing a database with multiple eCommerce
websites pertaining to different types of customers (i.e., B2B and B2C).

- :guilabel:`Tax Excluded`: the price displayed on the website is **tax-excluded** and the tax is
  computed at the cart-review step;
- :guilabel:`Tax Included`: the price displayed on the website is **tax-included**.

You can choose to display the type of pricing next to the product price 
by going to
:menuselection:`Website app --> Site --> Homepage --> Shop`, selecting one of your product, and then
:menuselection:`Edit --> Customize tab` and enabling :guilabel:`Tax Indication`.

.. image:: price_management/price-tax-display-type.png
   :align: center
   :alt: Tax type displayed on the product page

```

Reply:
Not sure what was changed here ? Your suggestion is identical to my content